### ISSUE TITLE
로그인 구현 변경 (쿠키 → 세션)

### ISSUE THREAD

**Issue by naamoonoo** [woowa-techcamp-2020/market-6#27]
### TL;DR

Change cookie(token)-based authentication to session.

### Manifesto

At first we planned to host servers for each server side and client side applications. This way provides great flexibility, independency and improves user experience. If one dies, one survives, vice versa. Nevertheless, the **market-6** is a small, short-period project where the advantages of running each micro service independently may not effectively enhance the system. Also we have to implement token system to communicate the authentication between the servers, but we're running out of time. So we finally decided to run a single server that hosts both APIs and web pages.
